python/ray/data/_internal/execution/backpressure_policy/concurrency_cap_backpressure_policy.py

```diff
@@ -17,66 +19,28 @@ class ConcurrencyCapBackpressurePolicy(BackpressurePolicy):
     """A backpressure policy that caps the concurrency of each operator.
 
     The concurrency cap limits the number of concurrently running tasks.
-    It will be set to an intial value, and will ramp up exponentially.
 
     The concrete stategy is as follows:
-    - Each PhysicalOperator is assigned an initial concurrency cap.
+    - Each PhysicalOperator is assigned a concurrency cap.
     - An PhysicalOperator can run new tasks if the number of running tasks is less
       than the cap.
```

Comment:
The comment looks a bit too verbose since we've removed the exponential ramp-up. We can simplify it to something like "The policy will limit the number of concurrently running tasks based on its `concurrency` parameter.`